ansible_collections/arista/avd/docs/contribution/authoring_eos_cli_config_gen.md

```diff
@@ -0,0 +1,111 @@
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Contribution Guide for the `eos_cli_config_gen` role
+
+This document outlines the steps and checklist for contributing to the `eos_cli_config_gen` role under Arista AVD.
+
+## Steps to add a new feature to the `eos_cli_config_gen` role
+
+### Prepare development environment
+
+Follow the [Development Tooling Guide](https://avd.arista.com/5.0/docs/contribution/development-tooling.html).
```

Comment:
So two things
1. use stable
2. I would think you could use a relative link there instead of absolute

```suggestion
Follow the [Development Tooling Guide](https://avd.arista.com/stable/docs/contribution/development-tooling.html).
```

same for all other links